src/Parser/MarkdownParser.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Codelicia\Xulieta\Parser;
+
+use Codelicia\Xulieta\External\Markinho;
+use Codelicia\Xulieta\ValueObject\SampleCode;
+use Symfony\Component\Finder\SplFileInfo;
+
+class MarkdownParser implements Parser
+{
+    public function isValid(SplFileInfo $file) : bool
+    {
+        return true;
```

Comment:
Shouldn't we verify for something or this is already being verified?

Reply:
In markdown is not really possible to be invalid. At least not as XML, HTML, RST, etc.

Reply:
Nice review! It may be not clear but that piece of code does the markdown syntax validation, not the PHP validation.

There is no way we can think of that a markdown could be invalid. It can get rendered in a wrong and weird way but there is no syntax error as far as we know.

Reply:
Nice thing, thanks for the answer <3